### PR TITLE
feat: remove upsert from generate schedule

### PR DESCRIPTION
**Pull request by aomi**
Had issues with generate schedule and unique constants failing. This only allows connecting with existing courses.

### CODE REVIEW

File: src/prisma/course.ts [hunk 1]
```diff
@@ -119,70 +117,39 @@ async function upsertCourses(
...
-          user: {
-            connect: {
-              id: prof?.id,
-            },
+      course: {
+        connect: {
+          subject_courseCode_term: {
+            courseCode: course.courseNumber,
+            subject: course.subject,
+            term: term,
```
Comment (kendajani) [inline src/prisma/course.ts:149]: Would this make more sense as `connectOrCreate` in the case that there is no course with these parameters?

Comment (aomi) [reply src/prisma/course.ts:149]: We want to reject them as there shouldn't be unknown courses being scheduled at this point. The list of courses we use to seed to from frontend so we're matching in terms of courses we're dealing with now.

Comment (kendajani) [reply src/prisma/course.ts:149]: Sounds good

**Review by kendajani — APPROVED** [review]
LGTM

**Pull request merged**